api/v1_create_reward_code.go

```diff
@@ -0,0 +1,126 @@
+package api
+
+import (
+	"context"
+	"crypto/ed25519"
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"math/big"
+
+	"api.audius.co/config"
+	"github.com/gagliardetto/solana-go"
+	"github.com/gofiber/fiber/v2"
+	"github.com/jackc/pgx/v5"
+)
+
+const (
+	signedMessage = "code"
```

Comment:
nit: variable name here is a tad generic, I had to jump and forth a few times to figure out how it was used :-)